### DISCUSSION TITLE
How can I change the certificate type if I request or renew it?

### DISCUSSION THREAD

**Discussion by Chrishan68** [NginxProxyManager/nginx-proxy-manager#2658] [Q&A]
Dear all,

the request or the renewal process for my certificates are working fine in npm and I also found the folder where the pem files are stored.
I am also able to convert the pem files with openssl to crt or pfx certificates with the private key. But they are not compatible with Microsoft Exchange. Therefore I want to change the key size, public key, public key parametes and key usage. But where can I find the config file for this and which parameter names and values are valid?

Thanks in advance!!!

**Comment by bmmmm** [comment]
as far as I am aware of, this is only configurable inside the docker / certbot.
see: [https://eff-certbot.readthedocs.io/en/stable/using.html#id20](https://eff-certbot.readthedocs.io/en/stable/using.html#id20) and https://eff-certbot.readthedocs.io/en/stable/using.html#id36

**Comment by Chrishan68** [comment]
Hi
I found it on my own!
You have to edit this file => 
![image](https://user-images.githubusercontent.com/76147801/224921244-2429eee0-f7b3-45b0-afb7-bbbaca070515.png)
